composite/token-allocation/src/adapter.ts

```diff
@@ -35,19 +36,34 @@ export const marketCapTotalValue = (
     .toNumber()
 }
 
-const inputParams = {
-  allocations: true,
-  quote: false,
-}
-
 const toValidAllocations = (allocations: TokenAllocations): TokenAllocations => {
```

Comment:
We don't have `TokenAllocations` on input, we have `any` that we need to validate:

 ```suggestion
const toValidAllocations = (allocations: any): TokenAllocations => {
```